lib/pbench/test/unit/agent/task/test_move_results.py

```diff
@@ -125,18 +108,32 @@ def test_move_results(monkeypatch, caplog, pytestconfig):
 
         caplog.set_level(logging.DEBUG)
 
+        responses.add(
+            responses.GET,
+            "http://pbench.example.com/api/v1/host_info",
+            status=200,
+            body="pbench@pbench-server:/srv/pbench/pbench-move-results-receive/fs-version-002",
+        )
+        responses.add(
+            responses.PUT,
+            f"http://pbench.example.com/api/v1/upload/{script}_{config}_{date}.tar.xz",
+            status=200,
+        )
+
         runner = CliRunner(mix_stderr=False)
         result = runner.invoke(
-            results.pmr,
+            move.main,
             args=[
                 TestMoveResults.CTRL_SWITCH,
                 TestMoveResults.CTRL_TEXT,
-                TestMoveResults.USER_SWITCH,
-                TestMoveResults.USER_TEXT,
-                TestMoveResults.PRFX_SWITCH,
-                TestMoveResults.PRFX_TEXT,
+                TestMoveResults.TOKN_SWITCH,
+                TestMoveResults.TOKN_TEXT,
             ],
-            input=f"{TestMoveResults.TOKN_TEXT}\n",
         )
-        assert result.exit_code == 0
-        assert result.stdout_bytes
+        assert (
+            result.exit_code == 0
+        ), f"Expected a successful operation, exit_code == {result.exit_code:d}"
```

Comment:
Yes, I'll fix this.

BUT ...

This brings up another problem with the `CliRunner` ... it is completely hiding stack traces.

I had to resort to adding my own `traceback` module invocation to get them printed to `stdout` to debug problems.

I think this is because our default agent logging is to the `pbench.log` file, but not entirely sure.

Reply:
Tracked in #2335.